autoload/codefmt.vim

```diff
@@ -393,3 +393,21 @@ endfunction
 function! codefmt#DisableIsAvailableChecksForTesting() abort
   let s:check_formatters_available = 0
 endfunction
+
+
+""
+" @private
+" Undoes the effect of @function(#DisableIsAvailableChecksForTesting), allowing
+" FORMATTER.IsAvailable checks to work normally.
+function! codefmt#DontDisableIsAvailableChecksForTesting() abort
+  let s:check_formatters_available = 1
+endfunction
```

Comment:
Any suggestions for a name for the single method? `codefmt#OverrideWhetherToDoIsAvailableChecksForTesting({enable})`? Long names should be fine for this as long as they're clear at a glance.